subprojects/build-init/src/main/java/org/gradle/buildinit/plugins/internal/BuildScriptBuilder.java

```diff
@@ -346,6 +347,16 @@ public BuildScriptBuilder block(@Nullable String comment, String methodName, Act
         return this;
     }
 
+    public BuildScriptBuilder javaLanguageVersion(JavaLanguageVersion languageVersion) {
```

Comment:
:lipstick: for explicitness: this is about configuring a java toolchain for a given language version
```suggestion
    public BuildScriptBuilder javaToolchainFor(JavaLanguageVersion languageVersion) {
```